lib/ansible/galaxy/dependency_resolution/dataclasses.py

```diff
@@ -564,6 +564,15 @@ def is_concrete_artifact(self):
     def is_online_index_pointer(self):
         return not self.is_concrete_artifact
 
+    @property
+    def is_pinned(self):
+        """Indicate if the version set is considered pinned."""
```

Comment:
Might be good to expand on what `pinned` actually means in this context.

Reply:
Great suggestion, will do!